deployment/rdma-cni-daemonset-pre-1.16.yaml

```diff
@@ -8,22 +8,22 @@ metadata:
     tier: node
     app: rdma-cni
 spec:
+  updateStrategy:
+    type: RollingUpdate
```

Comment:
why this change?
Also I am not sure we need the pre1.16 kubernetes. It don't have topology manager so none will use SR-IOV with max performance

Reply:
Thats how multus does it. see : https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/

the tl;dr; is : with Rolling update you dont need to delete the pods when you update the daemonset

Reply:
the pre-1.16 can be removed if not needed, but we will need to update the README to state that kubernetes version is expected to be >= 1.16

Reply:
Ok let do that 1.16 and above